.github/workflows/ci.yml

```diff
@@ -185,8 +183,6 @@ jobs:
     steps:
     - name: Checkout repository
       uses: actions/checkout@v1
-      with:
-        fetch-depth: 1
```

Comment:
It looks like this fixed CI. Not sure why, but fine to leave this as is. (And this should be in a separate commit.)